content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
+linktitle: Crosswalk for Kubernetes
+menu:
+  userguides:
+    parent: guides
+    identifier: crosswalk-kubernetes
+
+aliases: ["/docs/reference/crosswalk/kubernetes/"]
+---
+
+Pulumi Crosswalk for Kubernetes is a collection of industry standard
+best-practices and tasks for managing Kubernetes and its infrastructure in production.
+
+In this guide we will show you, using batteries-included examples, how to provision
+and use these technologies together in a way that maintains high release velocity, without
+sacrificing security, governance, or stability.
+
+This guide is for provisioning and configuring production-grade Kubernetes
+clusters. If you are just getting started with Pulumi and Kubernetes, the [Get
+Started guide][k8s-get-started] is a better place to start.
+
+<center><img src="/images/docs/quickstart/kubernetes/cake.svg" width="670"></center>
+
+## Getting Started
+
+* Introduction
+    * [Get Started with Kubernetes][k8s-get-started]
+    * [Production Architecture for Teams]({{< relref "/docs/guides/crosswalk/kubernetes/introduction/01-architecture.md" >}})
```

Comment:
What you have is fine, but you could leave off the `.md` if you wanted.

```suggestion
    * [Production Architecture for Teams]({{< relref "/docs/guides/crosswalk/kubernetes/introduction/01-architecture" >}})
```

Reply:
Done